Makefile

```diff
@@ -180,14 +180,14 @@ macos-setup:
 
 # push steps require auth on docker hub
 docker-toolchain:
-	@if git diff-index --quiet HEAD ; then \
-		docker build --no-cache --file dockerfiles/Dockerfile-toolchain --tag codaprotocol/coda:toolchain-$(GITLONGHASH) . && \
+	@# if git diff-index --quiet HEAD ; then
+		docker build --file dockerfiles/Dockerfile-toolchain --tag codaprotocol/coda:toolchain-$(GITLONGHASH) . && \
 		docker tag  codaprotocol/coda:toolchain-$(GITLONGHASH) codaprotocol/coda:toolchain-latest && \
 		docker push codaprotocol/coda:toolchain-$(GITLONGHASH) && \
 		docker push codaprotocol/coda:toolchain-latest ;\
-	else \
-		echo "Repo has uncommited changes, commit first to set hash." ;\
-	fi
+	#else \
+		#echo "Repo has uncommited changes, commit first to set hash." ;\
+	#fi
```

Comment:
Now that you've presumably committed the docker changes, is commenting out these conditionals still necessary?

Reply:
oops, thanks